app/views/course/assessment/submissions/_submission.html.slim

```diff
@@ -4,7 +4,7 @@
   td = link_to(format_inline_text(assessment.title),
                course_assessment_path(current_course, assessment))
   td = format_datetime(submission.submitted_at) if submission.submitted_at
-  td = Course::Assessment::Submission.human_attribute_name(submission.workflow_state)
+  td style="text-align: center;" = Course::Assessment::Submission.human_attribute_name(submission.workflow_state)
```

Comment:
The correct thing is not to inline styles here, rather add a css selector for center text alignment. Bootstrap offers something similar with `.text-center` I think.

Reply:
If fact, you can experiment declaring the css selector on the entire table to see if that works rather than applying it on each element.

Reply:
@benleong only wants the center alignment for specific columns.

Reply:
Then use a class and css to style, and not inline the styles here.